### ISSUE TITLE
Add 'scope="col"' to column headings.

### ISSUE THREAD

**Issue by JeffJacobson** [WSDOT-GIS/arcgis-section-508-table-viewer#1]
[The a11y project accessible data table guide](https://a11yproject.com/posts/accessible-data-tables/) states that column headings (the `<th>` element) should have a `scope` attribute set to `col` to let the screen reader know that the heading applies to the entire column.
